### DISCUSSION TITLE
Negative indents

### DISCUSSION THREAD

**Discussion by iqnite** [TodePond/GulfOfMexico#563] [Q&A]
It's cool that DreamBerd e/acc supports negative indents. But what if you use positive and negative indents in the same code?
For example:
```
const const x = prompt("Enter a number: ")!
if (x > 0) {
   alert("You entered a positive number!")!
   if (x > 100) {
alert("You entered a big number!")?
   }
}
```
Here, the line ending with the question mark is indented negatively and is part of the `x > 100` check, but it may be interpreted as part of the top-level program.
I first thought that DreamBerd e/acc looks at the curly brackets, but why do we need indentation anyway if it does?

**Comment by mybearworld** [comment]
I think it looks at the indentation, as curly braces don't matter.

> I first thought that DreamBerd e/acc looks at the curly brackets, but why do we need indentation anyway if it does?

We don't *need* it per se, it just makes code more readable, so it's good that Dreamberd requires that.
